### ISSUE TITLE
Consistent Field Names 

### ISSUE THREAD

**Issue by sammysmallman** [ssilverman/rdm-schema#23]
Fields duplicated across parameters should use the same "name" value.
For example DEVICE_INFO contains the personality index, personality count and dmx start address, but uses different names for those fields rather than what is used in DMX_PERSONALITY and DMX_START_ADDRESS.

**Comment by ssilverman** [comment]
This is an issue with the specification and not the DEVICE_INFO.json file.

**Comment by sammysmallman** [comment]
I agree this is an issue with the specification and inconsistent verbiage. But we can't let that roll into code. The name is used to uniquely identify one parameters field from another and in this situation a user of the schemas may not only be trying to keep their localisation files DRY but using the unique id's to correlate duplicate fields across parameters.
